x-pack/docs/en/rest-api/security/create-role-mappings.asciidoc

```diff
@@ -77,10 +77,14 @@ _Exactly one of `roles` or `role_templates` must be specified_.
 `rules`::
 (Required, object) The rules that determine which users should be matched by the
 mapping. A rule is a logical condition that is expressed by using a JSON DSL.
-See  <<role-mapping-resources>>. 
+See  <<role-mapping-resources>>.
 
 ==== Role Templates
 
+NOTE: Role templates require corresponding scripting feature to be enabled.
+Otherwise, role mapping with role templates will fail to create. See
+<<allowed-script-types-setting>>.
```

Comment:
We never call out what scripting features are related to which types of templates so "corresponding" might be confusing here. What's more,  we never call out the fact that you can reference a stored script in a template by id in any of the examples. Maybe something like: 

```suggestion
NOTE: Role templates require the relevant scripting feature to be enabled, otherwise the creation of a role mapping with role templates will fail. See <<allowed-script-types-setting>>.
```

and add an example where we reference a stored script template by id ?